lib/enumerations/error.rb

```diff
@@ -0,0 +1,8 @@
+module Enumerations
+  module Error
+    class DuplicatedSymbol < RuntimeError; end
+    class MissingPrimaryKey < RuntimeError; end
+    class DuplicatedPrimaryKey < RuntimeError; end
+    class InvalidValue < RuntimeError; end
+  end
+end
```

Comment:
I'd structure this differently:

1. Usually gems have all of their errors inherit a base class, so it's easier to catch all gem specific errors.
2. They are usually not namespaced with 'Error' (though there are some gems that do it like this, like the twitter gem however there is no real benefit in namespacing them if we suffix them with `Error`).

I like the way the [stripe gem](https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/errors.rb) has it.

I'd do it like this:

```ruby

module Enumerations
  class Error < StandardError; end
  class DuplicatedSymbolError < Error; end
  class MissingPrimaryKeyError < Error; end
  class DuplicatedPrimaryKeyError < Error; end
  class InvalidValueError < Error; end
  end
end
```